### PR TITLE
Feature/copy

### PR DESCRIPTION
**Pull request by adamkudrna**
* Write text content
* Point out featured reference

### CODE REVIEW

File: src/data/cards.yml [hunk 1]
```diff
@@ -1,39 +1,52 @@
...
+        engine for PHP. With TwigYard you can also build prototypes and benefit from utilizing your
+        Twig templates in future development.
 
 -   title: Forms
     image: forms.svg
     text: >
-        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. A autem dolore doloremque nihil quo quos vel voluptatem.</p>
+        Everybody needs a&nbsp;contact form. Configure your own form and provide it with your
+        custom markup to make it look exactly as you want. Send forms to multiple email addresses
+        or save them to&nbsp;log.
```
Comment (mbohal) [inline src/data/cards.yml]: In the future forms might have the ability to do more then just email or log. Other handlers are likely to be added, so I would prefer something more open:
` Send forms to multiple email addresses or save them to&nbsp;log` => `Form submission is processed by one or more handlers. For example, the data can be emailed or logged or both.`

File: src/data/cards.yml [hunk 2]
```diff
@@ -1,39 +1,52 @@
...
-        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. A autem dolore doloremque nihil quo quos vel voluptatem.</p>
+        Most web developers are familiar with Twig — a&nbsp;flexible, fast and secure template
+        engine for PHP. With TwigYard you can also build prototypes and benefit from utilizing your
+        Twig templates in future development.
 
 -   title: Forms
     image: forms.svg
     text: >
-        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. A autem dolore doloremque nihil quo quos vel voluptatem.</p>
+        Everybody needs a&nbsp;contact form. Configure your own form and provide it with your
```
Comment (mbohal) [inline src/data/cards.yml]: Forms can be of other types then contact.
`Everybody needs a&nbsp;contact form.` => `Everybody needs a&nbsp;form.`

File: src/data/cards.yml [hunk 3]
```diff
@@ -1,39 +1,52 @@
...
-        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. A autem dolore doloremque nihil quo quos vel voluptatem.</p>
+        TwigYard doesn’t use database, it only relies on static YAML files and Twig templates.
+        There is no risk of SQL injection, neither execution of hostile PHP code.
 
 -   title: Twig templates
     image: templates.svg
     text: >
-        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. A autem dolore doloremque nihil quo quos vel voluptatem.</p>
+        Most web developers are familiar with Twig — a&nbsp;flexible, fast and secure template
+        engine for PHP. With TwigYard you can also build prototypes and benefit from utilizing your
```
Comment (mbohal) [inline src/data/cards.yml]: I would prefer to be a bit more general.
`With TwigYard you can also build prototypes and benefit from utilizing your Twig templates in future development.` => `The templates can be easily reused in your Drupal or Symfony apps. This is perfect for quick prototyping.`

File: src/data/cards.yml [hunk 4]
```diff
@@ -1,39 +1,52 @@
...
     text: >
-        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. A autem dolore doloremque nihil quo quos vel voluptatem.</p>
+        Creating a&nbsp;new TwigYard site is as easy as creating a&nbsp;new directory. All you
+        need to start is a&nbsp;fairly simple configuration YAML and a&nbsp;Twig template.
 
 -   title: Security
     image: security.svg
     text: >
-        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. A autem dolore doloremque nihil quo quos vel voluptatem.</p>
+        TwigYard doesn’t use database, it only relies on static YAML files and Twig templates.
```
Comment (mbohal) [inline src/data/cards.yml]: `…it only relies on static YAML files and Twig templates` => `…everything is in YAML files and Twig templates`

File: src/data/cards.yml [hunk 5]
```diff
@@ -1,39 +1,52 @@
...
+        There is no risk of SQL injection, neither execution of hostile PHP code.
```
Comment (mbohal) [inline src/data/cards.yml]: `…neither execution of hostile PHP code.` => `… and the overall attack surface is low.`

File: src/data/references.yml [hunk 6]
```diff
@@ -1,20 +1,30 @@
...
-    text: >
-        With TwigYard I stopped doing all that boring mechanical work on and on again, I can finally focus on what is unique on a project. I especially love that TwigYard generates a picture in any dimensions I need.
--   name: Jan Kryšpín
-    position: Frontend developer
-    image: jan.jpg
-    text: >
-        TwigYard allows me to create any website or its prototype in seconds and I can use all these fancy features without bothering backend deveopers.
 -   name: Martin Bohal
-    position: Senior developer
+    position: Senior developer, TwigYard creator
```
Comment (mbohal) [inline src/data/references.yml]: Could we remove the `senior` please? All creators are seniors :)

Comment (crishpeen) [reply src/data/references.yml]: I thought this one is based on age and not experience.

**Pull request merged**